dddwa/dddperth-website
# Updates voting page

First phase update for the voting page. Adds a new hero area and content
layout to make the page a little more interesting.

### Testing
* Use the testing panel to set voting to open
* View http://localhost:3000/vote
* See the new layout

### Note
There's more that's going to be done to this page but this is a good first step if we run out of time for that.
